typescript/agentkit/package.json

```diff
@@ -54,7 +57,8 @@
     "mock-fs": "^5.2.0",
     "ts-jest": "^29.2.5",
     "tsd": "^0.31.2",
-    "typescript": "^5.7.2"
+    "typescript": "^5.7.2",
+    "vitest": "^3.0.5"
```

Comment:
Please remove – we currently use Jest for testing, but open to other options in the future